examples/prometheus/prometheus.yaml

```diff
@@ -243,12 +252,22 @@ objects:
       - name: example-rules
         interval: 30s # defaults to global interval
         rules:
-        - alert: Node Down
+        - alert: NodeDown
           expr: up{job="kubernetes-nodes"} == 0
           annotations:
             miqTarget: "ContainerNode"
             severity: "HIGH"
-            message: "{{$labels.instance}} is down"
+            message: "{{$labels.instance}} is down"    
+        - alert: HangingBuild
+          expr: count(openshift_build_running_phase_start_time_seconds{} < time() - ${PROMETHEUS_ACTIVE_BUILD_ALERT_DURATION}) > ${PROMETHEUS_ACTIVE_BUILD_ALERT_THRESHOLD}
+          annotations:
+            severity: "MEDIUM"
+            message: "{{$labels.instance}} indicates at least ${PROMETHEUS_ACTIVE_BUILD_ALERT_THRESHOLD} OpenShift builds are taking more than ${PROMETHEUS_ACTIVE_BUILD_ALERT_DURATION} seconds to complete"
```

Comment:
This is a more clear metric.  However, I'm not sure I want to check the other two in until you prove that you want these alerts on.  I.e. when we upgrade to 3.7.0 I'll turn your proposed alerts on and send them to you.  Then you can decide whether they work or not.

Reply:
Based on what I've learned as part of taking on this card (see the referenced google doc), I'm inclined to not do any of these alerts, either in this example template or manually submitting them to you (if not submitting to you is even an option), and wait until we get more practical experience with the build subsystem in production.  But the card was assigned and initiating a PR to start discussions seemed the best thing to do.

On the values of the conditions / evil template parameters, for the current zabbix equivalent that ops is employing today, https://github.com/openshift/openshift-tools/blob/prod/scripts/monitoring/cron-send-stuck-builds.py#L60-L93, the "age" is not hard coded, but a variable passed in.

That said, I've re-initiated the email exchange I started with them during the build metrics work, and seeing if default values emerge (perhaps the calling to the referenced function hardcodes the value).

Reply:
Let's actually base it on real data.  Suggest your top 3 alerts based on recent bugs, and I'll turn them on after the 3.7.0 upgrade.

Reply:
10-4 re: alerts <=> recent bugs and https://github.com/openshift/origin/pull/16495#issuecomment-331299944 .... @bparees and I will start conferring on all that

for completeness, an update on what ops is currently doing with their zabbix stuff:
- they are currently concerned with builds stuck in NEW|PENDING ... not slow RUNNING builds
- I elaborated a bit that is really a pod scheduling issue, and might be captured by any new k8s or platform mgmt team metrics that get injected/rebased into openshift/origin
- but that we could add it to our build specific stuff easy enough if need
- their build alerts are at the lowest level; so a warning on the zabbix board, but nobody gets paged